packages/adblocker/test/utils.ts

```diff
@@ -119,3 +120,7 @@ export function getRequestSamplePath(url: string): string {
 export function loadRequestSample(path: string): string {
   return zlib.brotliDecompressSync(fs.readFileSync(path)).toString('utf8');
 }
+
+export function stringifyResource(distribution: ResourcesDistribution): string {
+  return JSON.stringify(distribution);
+}
```

Comment:
how about we rename `Resources.parse` to `Resources.fromDistribution` and move the JSON.parse out of it?

Reply:
Changed to [`c195453` (#4270)](https://github.com/ghostery/adblocker/pull/4270/commits/c1954539d0b00ba60d43c7e7e4b5d7aa3d9cc4cc)